src/defaults/definitions.js

```diff
@@ -38,5 +38,25 @@ module.exports = {
     period: 60,
     evaluationPeriods: 1,
     comparisonOperator: 'GreaterThanThreshold',
-  }
-};
\ No newline at end of file
+  },
+  bunyanWarnings: {
```

Comment:
In terms of flexibility: maybe it is more reasonable to define "general" metric that matches some user-defined pattern against logs versus having hardcoded ones?